indico/htdocs/sass/modules/_roles.scss

```diff
@@ -0,0 +1,90 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2017 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+.role-list-row .i-tag {
+    display: inline-flex;
+    justify-content: center;
+    align-items: baseline;
+}
+
+.i-tag:not(:first-child) {
+    margin-left: 0.2em;
+}
+
+.role-list-row {
+    background: #eee;
+}
+
+.role-list-row td {
+    padding: 0 !important;
+}
+
+.role-list-row td .content-area {
+    background: #fff;
+    margin: 0 0 0 58px !important;
+}
+
+.role-list-row td .content-area .empty {
+    padding: 10px;
+    font-style: italic;
+    color: $dark-gray;
+}
+
+.role-list-row td .content-area .toolbar {
+    padding: 10px;
+}
+
+.role-list {
+    padding: 10px;
+    box-sizing: border-box;
+    list-style-type: none;
+}
+
+.role-list li {
```

Comment:
since this is SASS now, it can be improved (nesting instaedof repeating  selectors all the time)